DedicatedServer/Scripts/Libs/Zrx/ModeLibs/Common/WarmUpReadyUp/WarmUpReadyUp_UI.Script.txt

```diff
@@ -1,18 +1,19 @@
+// #RequireContext CSmMlScriptIngame
+
 #Const C_ReadyEvent_Type	"ready"
 #Const C_ManialinkName		"WarmUpReadyUp"
 
 Text GetManialink() {
 	return """
 	<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
 	<manialink version="3" name="{{{C_ManialinkName}}}">
-		<frame id="player-ready-frame" pos="0 -60">
+		<frame id="player-ready-frame" pos="0 65">
+			<label pos="0 -8" z-index="0" size="40 5" text="pad  or  + " halign="center" textsize="4" textfont="GameFontRegular" valign="center2"/>
 			<quad pos="0 0" z-index="0" size="40 10" style="UICommon64_1" substyle="BgFrame1" halign="center" valign="center" colorize="000" opacity="0.75"/>
-
 			<quad pos="15 0" z-index="1" size="10 10" id="button-ready-background" style="UICommon64_1" substyle="BgFrame1" halign="center" valign="center" colorize="fff"/>
 			<label pos="15 0" z-index="2" size="10 10" id="button-ready-icon" text="" halign="center" valign="center" style="Default" textfont="GameFontBlack" textsize="4"/>
 			<quad pos="0 0" z-index="3" size="40 10" id="button-ready-activator" style="UICommon64_1" substyle="BgFrame1" halign="center" valign="center" opacity="0" scriptevents="1"/>
-
-			<label pos="-5 0" z-index="5" size="30 8" text="Ready" halign="center" valign="center" textsize="4" style="Default" textfont="GameFontSemiBold"/>
+			<label pos="-8.5 0" z-index="5" size="20.9 9.59" text="Ready" halign="center" textsize="4" style="Default" textfont="GameFontSemiBold" valign="center2"/><label pos="9 0" z-index="5" size="9.34 9.59" text="(F7)" halign="right" textsize="3" style="Default" textfont="GameFontRegular" valign="center2"/>
```

Comment:
Space it getting really cramped. Try reducing the text size. Also could try moving the F7 below the button, in the same format as the pad buttons shown above.